src/System.Drawing.Primitives/src/PinvokeAnalyzerExceptionList.analyzerdata

```diff
@@ -0,0 +1 @@
+user32.dll!GetSysColor
```

Comment:
@danmosemsft is this safe? We need this as System.Drawing.Color doesn't support Windows system colours until this PR.